specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -221,6 +221,9 @@ model AcsSmsReceivedEventData extends AcsSmsEventBaseProperties {
   /** The time at which the SMS was received */
   @madeRequired(ServiceApiVersions.v2024_01_01)
   receivedTimestamp: utcDateTime;
+
+  /** Number of segmensts in the message */
+  segmentCount?: int32;
```

Comment:
to confirm, this property is optional (i.e not always returned by the service as a part of the json payload)?

Reply:
Yes